src/process_utility.c

```diff
@@ -2734,7 +2735,13 @@ process_refresh_mat_view_start(ProcessUtilityArgs *args, Node *parsetree)
 	PopActiveSnapshot();
 	CommitTransactionCommand();
 
-	ts_cm_functions->continuous_agg_materialize(materialization_id, true);
+	cagg_fullrange = ts_cm_functions->continuous_agg_materialize(materialization_id, true);
+	if (!cagg_fullrange)
+	{
+		elog(WARNING,
+			 "REFRESH did not materialize the entire range since it was limited by "
```

Comment:
add a "the" after "by" so "REFRESH did not materialize the entire range since it was limited by the max_interval_per_job setting"